app/views/uploads/index.html.erb

```diff
@@ -1,31 +1,37 @@
-<%= render 'shared/organization_header' %>
+<%= render 'shared/layout_upload_page' do %>
 
-<div class="container">
-  <h1>Uploads</h1>
+  <div class="d-flex align-items-center">
+
+    <h2 class="font-weight-normal mb-0">Uploads</h2>
+
+    <div class="ms-auto p-2">
+      <span class="ms-4"><%= link_to 'New upload', new_organization_upload_path(@stream.organization, stream: (@stream.default? ? nil : @stream.friendly_id)), class: 'btn btn-primary' if can? :create, @stream.uploads.build %></span>
```

Comment:
I'm a little concerned about providing an upload button here. How do we know which stream the upload will be associated with?

Reply:
There's a similar issue on the upload show page. I'd be okay merging this as-is and handling the ambiguity of the "New upload" button as a separate design issue.

Reply:
OK. This was in the mockup so I added the button.
I reused some existing code that determines the stream here:
` stream: (@stream.default? ? nil : @stream.friendly_id))`

This New upload button code is the same all over the app, including on the Provider home page. The route `http://localhost:3000/organizations/stanford/uploads/new` is the upload page, and has no indication of what stream you're in.

Reply:
I created a new issue for this so we don't have to worry about for this PR: https://github.com/pod4lib/aggregator/issues/781